math/maxima/Portfile

```diff
@@ -19,30 +18,62 @@ long_description \
         tensors. Maxima yields high precision numeric results \
         by using exact fractions, arbitrary precision integers, \
         and variable precision floating point numbers. Maxima \
-        can plot functions and data in two and three dimensions.
+        can plot functions and data in two and three dimensions. \n\
+        Maxima includes the following commands: \n\
+        * \"rmaxima\" is a front end which provides advanced line-editing \
+        facilities via rlwrap. \n\
+        * \"xmaxima\" is a graphical interface for maxima, written in Tcl/Tk.
 
 homepage        http://maxima.sourceforge.net/
-master_sites    sourceforge
 
-checksums       md5     3bccafb512ff1054be700c61601088af \
-                sha256  f58ff98013fcaa37d8661b941672c38db58f982f082d6cfcca1f1b5fac7d71b3 \
-                rmd160  3c4b153554c9b9cb3442e030d503fbe975a614fc
+subport maxima-devel {
+    conflicts   maxima
+
+    # git describe --tags : branch-5_41-base-415-g7e71fc4
+    # Date:  Fri Jun 15 17:26:53 2018 -0500
+    # commit 7e71fc4b6a95cab6418227754f01fc44ac845da1
+    version     5.41post
```

Comment:
Not as comments, you should add it into `version`, e.g. `5.41-415-g7e71fc4`, `5.41-dev-20180615`, `5.41-r2018-06-15`, etc.

Reply:
Oh I see. I will changed '5.41post' to '5.41-dev-20180615'.